ee/fleet-agent-downloader/api/controllers/entrance/signup-sso-user-or-redirect.js

```
@@ -0,0 +1,55 @@
+      // If we did not find a user in the database for this SSO user, we'll create a new one.
+      let newUserRecord = await User.create({
+        fullName: ssoUserInfo.name,
+        emailAddress: ssoUserInfo.email,
+        password: await sails.helpers.passwords.hashPassword(ssoUserInfo.sub),// Note: this password cannot be changed.
+      }).fetch();
+      this.req.session.userId = newUserRecord.id;
+    }
+    // Redirect the logged-in user to the homepage.
+    return this.res.redirect('/');
+
+  }
```

ESLint

[error] Expected method shorthand.

(object-shorthand)